File: TravelAgent/Model/OfferRequest.cs
namespace TravelAgent.Model
{
    public class OfferRequest
    {
        public int Id { get; set; }
        public double MaxPrice { get; set; }
        public string DepartureLocation { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
        public TransportationType TransportationType { get; set; }
        public List<Location> Locations { get; set; }
        public int SpotNumber { get; set; } //za koliko ljudi je ponuda
    }
}

File: TravelAgent-Ng/src/app/models/offer-request.ts
export class OfferRequest {
    id: number = 0;
    departureLocation: string = '';
    description: string = '';
    startDate: string = '';
    endDate: string = '';
    maxPrice: number = 0.0;
    transportationType: string = '';
    locations: Array<string> = [];
    spotNumber: number = 0;
    clientId: number = 0;
}